shared/navLinks/links.ts

```diff
@@ -59,6 +59,12 @@ export const links = [
         href: "/use/ecosystem",
         image: "/images/nav/use-ecosystem.svg",
       },
+      {
+        title: messages.cliTitle,
+        description: messages.cliDescription,
+        href: "/use/get-started/installation",
+        image: "/images/nav/use-submarine.svg",
+      },
```

Comment:
Do we want this section to be last in the dropdown?

Reply:
I have no strong opinion. If you have a better recommendation of ordering I'm all ears. I think it's a pretty technical item so I don't think it matters where it is but people will look for it.

Reply:
Moved it in the ordering.